domains/_discord.azhammakestrash.is-a.dev.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "AzhamProdLive",
+        "email": "websitedomain.regroup588@passinbox.com"
```

Comment:
you need to add a comma at the "email" line at the end

Reply:
Done